### PR TITLE
CRO-463 Tracking of a particular widget on Home page and toggle-off live-price feature

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

To have a tracking on on widgets and liveprice features

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by aswathy-deriv — APPROVED** [review]

File: src/features/components/quill/layout-overlay/live-chat-button/index.tsx [hunk 1]
```diff
@@ -115,7 +116,13 @@ const LiveChatButton = () => {
             {is_livechat_interactive ? (
                 <FloatingButton
                     className="gtm-deriv-livechat"
-                    onClick={() => LC_API.current.open_chat_window()}
+                    onClick={() => {
```
Comment (habib-deriv) [inline src/features/components/quill/layout-overlay/live-chat-button/index.tsx:119]: Can we have function in utility and use it here. Would be good for re-usabilty

File: src/features/components/quill/layout-overlay/live-chat-button/index.tsx [hunk 2]
```diff
@@ -115,7 +116,13 @@ const LiveChatButton = () => {
...
+                        LC_API.current.open_chat_window()
+                        Analytics.trackEvent('ce_widget_usage_form', {
```
Comment (habib-deriv) [inline src/features/components/quill/layout-overlay/live-chat-button/index.tsx:121]: Same here

File: src/features/components/quill/layout-overlay/whats-app-button/index.tsx [hunk 3]
```diff
@@ -9,7 +10,15 @@ const WhatsappButton = () => {
     const { is_mobile } = useBreakpoints()
 
     return (
-        <FloatingButton onClick={() => window.open(whatsapp_url, '_blank')}>
+        <FloatingButton
+            onClick={() => {
+                Analytics.trackEvent('ce_widget_usage_form', {
+                    action: 'click',
+                    widget_name: 'whatsapp',
```
Comment (habib-deriv) [inline src/features/components/quill/layout-overlay/whats-app-button/index.tsx:17]: Same here.Widget name can be passed via argument

**Review by habib-deriv — CHANGES_REQUESTED** [review]
Added some comments

File: src/features/components/quill/layout-overlay/live-chat-button/index.tsx (hunk 1 above)
Comment (bio-crow) [reply src/features/components/quill/layout-overlay/live-chat-button/index.tsx:119]: it's possible. I moved that function to common folder

Comment (NikitK-deriv) [reply src/features/components/quill/layout-overlay/live-chat-button/index.tsx:119]: yes, it's possible. I add convert that function to ulility one and move to /common folder

**Review by habib-deriv — DISMISSED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**